docs/theme/scripts.html

```diff
@@ -8,7 +8,7 @@
 
  // configure UI components
  window.grvConfig = window.grvConfig || {
-      docVersions:  ["3.1", "3.2", "4.0", "4.1", "4.2"],
+      docVersions:  ["1.3", "2.0", "2.3", "2.4", "2.5", "2.7", "3.0", "3.1", "3.2", "4.0", "4.1", "4.2", "4.3"],
```

Comment:
Why are we adding these older versions back in?

Reply:
good catch, I've fixed this.